src/components/Card.js

```diff
@@ -127,6 +127,15 @@ export const Card = () => {
               <input
                 type="checkbox"
                 id="hide-password-checkbox"
+                disabled={
```

Comment:
This logic should be factored out into a function instead of in-lined in the markup:

```js
const disableHidePassword = () => { ... }

<input
  disabled={disableHidePassword()}
```

Reply:
Refactored. 👍